code/modules/surgery/organs/heart.dm

```diff
@@ -27,6 +68,9 @@
 
 /obj/item/organ/heart/attack_self(mob/user)
 	..()
+	if(assembly)
+		assembly.attack_self(user)
```

Comment:
pretty sure there's a signal for this too

Reply:
COMSIG_ITEM_ATTACK_SELF